internal/relocated/relocated_bundle.go

```diff
@@ -0,0 +1,32 @@
+package relocated
```

Comment:
nit: file could be renamed to `bundle.go`, as `relocated/relocated_bundle.go` has kind of repetition 😄